packages/react-composites/src/composites/CallWithChatComposite/Strings.tsx

```diff
@@ -45,6 +45,14 @@ export interface CallWithChatCompositeStrings {
    * Only used with `mobileView` set to `true`.
    */
   moreDrawerSpeakerMenuTitle: string;
+  /**
+   * {@Link CallWithChatComposite} Mobile Control bar More drawer label for narrator
```

Comment:
The comment doesn't seem to match the field name / intent. This is also the label that is shown on the UI, right?

Reply:
Right, I labelled it in the context I was working on when its really a side effect! will change this to be more appropriate 😅